pymc/gp/hsgp_approx.py

```diff
@@ -218,12 +296,13 @@ def __init__(
 
         self._drop_first = drop_first
         self._m = m
-        self._m_star = int(np.prod(self._m))
+        self._m_star = self.n_basis_vectors = int(np.prod(self._m))
```

Comment:
is this double `=`intentional?

Reply:
Yes, it's a double assignment (if I remember the technical term correctly)

Reply:
ok! just double checking ;)